API/MoodisAPI/apiMoodis/apiMoodis/Controllers/EmotionController.cs

```diff
@@ -0,0 +1,85 @@
+﻿using apiMoodis.Models;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Net;
+using System.Net.Http;
+using System.Web.Http;
+using System.Data.Entity;
+
+namespace apiMoodis.Controllers
+{
+    public class EmotionController : ApiController
```

Comment:
Might be worth to add method which returns emotion by imageId as well

Reply:
This API was meant more as a guideline that contains basic CRUD operations rather than a fully completed service, because I believe that during the process of changing our app code more functions such as these will be needed. One of them I was sure of - GetByUsernameUser. Nonetheless I will add it right away